public/keymaps/p/pom_keyboards_tnln95_default_full.json

```diff
@@ -0,0 +1,22 @@
+{

```

Comment:
This file should be deleted; QMK Configurator only supports one keymap per keyboard at this time, and has no capability to use this file.